airbyte-webapp/src/core/domain/connector/types.ts

```diff
@@ -4,6 +4,15 @@ import {
 } from "core/domain/connection";
 import { DestinationSyncMode } from "core/domain/catalog";
 
+export enum releaseStage {
+  "ALPHA" = "alpha",
+  "BETA" = "beta",
+  "GENERALLY_AVAILABLE" = "generally_available",
+  "CUSTOM" = "custom",
+  "RELEASE" = "release",
+  "NEW_RELEASE" = "new_release",
```

Comment:
The release stage enum in backend only has 4 values: https://github.com/airbytehq/airbyte/blob/master/airbyte-api/src/main/openapi/config.yaml#L2531-L2534: Alpha, Beta, Custom, Generally_available. I don't think `release` and `new_release` here should exist.

Reply:
Removed `release` and `new_release` statuses